ansible/configs/ibm-rhoic/destroy_env.yml

```diff
@@ -137,7 +137,7 @@
       uri:
         url: "{{ ibm_cloud_api_subnet_url }}/{{ rhoic_subnet_id }}?{{ ibm_cloud_api_query_params }}"
         method: DELETE
-        status_code: 204,409
+        status_code: 204,409,500,502,-1
```

Comment:
I have not verified in the API docs, but this should be only 204 for success.

Reply:
204 for success, but I have seen it twice where it returned a 500 but the next retry reported a 409 (duplicate) as it created it in the background but failed to report. So that is why this will have two success codes.

Reply:
yup